package.json

```diff
@@ -74,6 +74,7 @@
     "sha256-file": "1.0.0",
     "sql.js": "pastelnetwork/sql.js#230351f",
     "squoosh": "pastelnetwork/squoosh#d44b1b4",
+    "styled-components": "^5.3.0",
```

Comment:
Could you please specify the exact version of the package? This would prevent unwanted updates, and this is the pattern we're following

Reply:
Sure, changed.